js_tests/admin/URLify.test.js

```diff
@@ -7,10 +7,6 @@ QUnit.test('empty string', function(assert) {
     assert.strictEqual(URLify('', 8, true), '');
 });
 
-QUnit.test('strip nonessential words', function(assert) {
-    assert.strictEqual(URLify('the D is silent', 8, true), 'd-silent');
```

Comment:
WDYT about adapting these tests rather than deleting them? Adapting them would show -- through a test -- that stop words aren't treated as a special. It would need the test title updated too.

Reply:
Yes we should adopt existing tests.